query/src/interpreters/interpreter_copy_v2.rs

```diff
@@ -0,0 +1,315 @@
+use std::path::Path;
```

Comment:
This looks wierd.

Reply:
Oh, fixed in https://github.com/datafuselabs/databend/pull/6074/commits/001861871a3152c4110e161ac26a36a532bdfc55